src/Symfony/Component/Lock/Lock.php

```diff
@@ -75,7 +75,7 @@ public function acquire($blocking = false)
             return false;
         } catch (\Exception $e) {
             $this->logger->warning('Failed to lock the "{resource}".', array('resource' => $this->key, 'exception' => $e));
-            throw new LockAcquiringException('', 0, $e);
+            throw new LockAcquiringException('Failed to lock the resource.', 0, $e);
```

Comment:
Fixed,
For consistency, I use the format already defined in other log messages ` ... the "{resource}" lock`.
And I fixed some inconsistent logs message which didn't use the same vocabulary.